tsconfig.app.json

```diff
@@ -18,7 +18,13 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+
+    /* Import alias */
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
Thank you @waffensultan! I believe we need the following in our vite.config.ts as well?

```diff
+ import path from 'path';
export default defineConfig({
  resolve: {
    alias: {
+      '@': path.resolve(__dirname, './src'),
    },
  },
  plugins: [react(), tailwindcss()],
});
```